features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/ElasticFlowRepository.java

```diff
@@ -114,6 +114,21 @@
     private final ClassificationEngine classificationEngine;
 
     private final int bulkRetryCount;
+    
+    private static String eventIndexName = "netflow";
+    
+    //public static void setEventIndexName(String eventIndexName) {
+	//ElasticFlowRepository.eventIndexName = eventIndexName;
+	//}
+    
+    public String getModifiedIndex()
+    {
+    	Objects.requireNonNull(eventIndexName, "null value");
```

Comment:
The non-null check needs to be in the setter, not the getter. Also you can omit the message "null value" as it doesn't add any context.